### PR TITLE
[release/1.13.0rc0] Add release logs

### PR DESCRIPTION
**Pull request by krfricke**
## Why are these changes needed?

Preliminary release logs for review and approval.

cc @rkooo567 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/release_logs/1.13.0rc0/benchmarks/many_actors.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "actors_per_second": 298.0171917593177,
+    "num_actors": 10000,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "actors_per_second",
+            "perf_metric_type": "THROUGHPUT",
+            "perf_metric_value": 298.0171917593177
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/benchmarks/many_actors.json:10]: @jjyao @scv119 IT was used to be 388, so there was about 30% regression. Was there any perf alert before?

File: release/release_logs/1.13.0rc0/benchmarks/many_nodes.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "num_tasks": 1000,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "tasks_per_second",
+            "perf_metric_type": "THROUGHPUT",
+            "perf_metric_value": 8.016102950431257
+        }
+    ],
+    "success": "1",
+    "tasks_per_second": 8.016102950431257,
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/benchmarks/many_nodes.json:13]: LGTM

File: release/release_logs/1.13.0rc0/benchmarks/many_pgs.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "num_pgs": 1000,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "pgs_per_second",
+            "perf_metric_type": "THROUGHPUT",
+            "perf_metric_value": 15.153418743743368
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/benchmarks/many_pgs.json:9]: LGTM

File: release/release_logs/1.13.0rc0/benchmarks/many_tasks.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "num_tasks": 10000,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "tasks_per_second",
+            "perf_metric_type": "THROUGHPUT",
+            "perf_metric_value": 28.076234123655464
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/benchmarks/many_tasks.json:9]: LGTM

File: release/release_logs/1.13.0rc0/microbenchmark.json [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+    "1_1_async_actor_calls_with_args_async": [
+        2384.670664299169,
+        152.82405396732784
+    ],
+    "1_n_actor_calls_async": [
+        10671.14624877688,
+        526.5357298149904
+    ],
+    "1_n_async_actor_calls_async": [
+        11120.569744924334,
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/microbenchmark.json:31]: Same for this.

File: release/release_logs/1.13.0rc0/microbenchmark.json [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+    "multi_client_put_calls_Plasma_Store": [
+        9745.670014416988,
+        126.20971804569032
+    ],
+    "multi_client_put_gigabytes": [
+        36.85514798847486,
+        1.6722498011347091
+    ],
+    "multi_client_tasks_async": [
+        37237.079715148226,
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/microbenchmark.json:75]: This also had pretty big change (46643 -> 37237). cc @jjyao for confirmation

File: release/release_logs/1.13.0rc0/microbenchmark.json [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+        777.4796877284789
+    ],
+    "n_n_actor_calls_async": [
+        34770.47668215435,
+        842.1881338319646
+    ],
+    "n_n_actor_calls_with_arg_async": [
+        2634.652949167727,
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/microbenchmark.json:83]: Same for this 3336 -> 2634

File: release/release_logs/1.13.0rc0/scalability/single_node.json [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 27.485386731000006
+        },
+        {
+            "perf_metric_name": "1000000_queued_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 156.974035497
+        },
+        {
+            "perf_metric_name": "107374182400_large_object_time",
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/scalability/single_node.json:32]: LGTM

File: release/release_logs/1.13.0rc0/stress_tests/stress_test_many_tasks.json [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+    ],
+    "stage_0_time": 6.338145971298218,
+    "stage_1_avg_iteration_time": 21.204884552955626,
+    "stage_1_max_iteration_time": 21.92381715774536,
+    "stage_1_min_iteration_time": 20.679417371749878,
+    "stage_1_time": 212.04893469810486,
+    "stage_2_avg_iteration_time": 56.038217830657956,
+    "stage_2_max_iteration_time": 56.81925582885742,
+    "stage_2_min_iteration_time": 54.01719784736633,
+    "stage_2_time": 280.19193506240845,
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/stress_tests/stress_test_many_tasks.json:42]: LGTM

File: release/release_logs/1.13.0rc0/stress_tests/stress_test_placement_group.json [hunk 10]
```diff
@@ -0,0 +1,17 @@
+{
+    "avg_pg_create_time_ms": 0.7913740735731964,
+    "avg_pg_remove_time_ms": 3.310559710209569,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "avg_pg_create_time_ms",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 0.7913740735731964
```
Comment (rkooo567) [inline release/release_logs/1.13.0rc0/stress_tests/stress_test_placement_group.json:8]: LGTM

**Review by rkooo567 — COMMENTED** [review]
Generally, there's some regression on microbenchmark performance (20%). Is this expected, or have we had perf alarmed? cc @jjyao and @scv119 for the confirmation.

File: release/release_logs/1.13.0rc0/benchmarks/many_actors.json (hunk 1 above)
Comment (jjyao) [reply release/release_logs/1.13.0rc0/benchmarks/many_actors.json:10]: The latest run in master is `467.168` (https://buildkite.com/ray-project/release-tests-branch/builds/476#302bbe89-8f43-4560-bca7-4a053b1a6f4d). Based on the history, I feel this is false alarm.

File: release/release_logs/1.13.0rc0/microbenchmark.json [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+    "1_1_async_actor_calls_sync": [
+        1760.6933323973485,
+        29.402298751865963
+    ],
+    "1_1_async_actor_calls_with_args_async": [
+        2384.670664299169,
+        152.82405396732784
+    ],
+    "1_n_actor_calls_async": [
+        10671.14624877688,
```
[the remark this answers is not in the document]
Comment (jjyao) [reply release/release_logs/1.13.0rc0/microbenchmark.json:27]: Based on the history, I feel this is also false alarm.

File: release/release_logs/1.13.0rc0/microbenchmark.json (hunk 6 above)
Comment (jjyao) [reply release/release_logs/1.13.0rc0/microbenchmark.json:75]: Based on the history, it varies a lot so a bit hard to tell. In 1.11.0 release log it's 
```
"multi_client_tasks_async": [
    38103.232324553144,
    4035.7281169130333
  ],
  ```

File: release/release_logs/1.13.0rc0/microbenchmark.json (hunk 7 above)
Comment (jjyao) [reply release/release_logs/1.13.0rc0/microbenchmark.json:83]: Based on the history, I feel this is false alarm as well.

**Comment by krfricke** [comment]
After the next cherry picks, we can run the microbenchmarks again and update this PR @rkooo567 @jjyao

File: release/release_logs/1.13.0rc0/microbenchmark.json (hunk 6 above)
Comment (krfricke) [reply release/release_logs/1.13.0rc0/microbenchmark.json]: Latest run 

```
    "multi_client_tasks_async": [
        37237.079715148226,
        39959.723850727976,
        777.4796877284789
        1564.815048000218
    ],
    ],
```

**Comment by krfricke** [comment]
Updated results after latest cherry-picks ptal @rkooo567 @jjyao

**Comment by krfricke** [comment]
Updated logs with final wheels results, ptal @rkooo567

**Comment by rkooo567** [comment]
Hey @krfricke should I review this?

File: release/release_logs/1.13.0/benchmarks/many_actors.json [hunk 12]
```diff
@@ -0,0 +1,15 @@
+{
+    "_peak_memory": 5.32,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n186\t1.61GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n2169\t0.88GiB\tpython distributed/test_many_actors.py\n211\t0.26GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n294\t0.12GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n76\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n404\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session auth_start\n2214\t0.06GiB\tray::MemoryMonitorActor.run()\n77\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=63d80251\n204\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.80.251:9031 --host=0.0.0.0\n244\t0.05GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/log_m",
+    "actors_per_second": 321.72392004133843,
```
Comment (rkooo567) [inline release/release_logs/1.13.0/benchmarks/many_actors.json:4]: Hmm, this has about 20% regression. However, when I checked the perf graph, this varies from 320 ~ 380. @krfricke can you double check if the previous run was faster than this?

File: release/release_logs/1.13.0/benchmarks/many_nodes.json [hunk 13]
```diff
@@ -0,0 +1,15 @@
+{
+    "_peak_memory": 4.24,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n211\t0.28GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n186\t0.16GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n2430\t0.16GiB\tpython distributed/test_many_tasks.py --num-tasks=1000\n287\t0.13GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n77\t0.12GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n404\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session auth_start\n78\t0.06GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=6795d173\n2475\t0.06GiB\tray::MemoryMonitorActor.run()\n204\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.94.43:9031 --host=0.0.0.0\n244\t0.05GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/log_m",
+    "num_tasks": 1000,
```
Comment (rkooo567) [inline release/release_logs/1.13.0/benchmarks/many_nodes.json:4]: LGTM

File: release/release_logs/1.13.0/benchmarks/many_tasks.json [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n186\t1.0GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n2121\t0.71GiB\tpython distributed/test_many_tasks.py --num-tasks=10000\n211\t0.46GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n287\t0.14GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n68\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n405\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session auth_start\n2166\t0.06GiB\tray::MemoryMonitorActor.run()\n69\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=cab3c8e8\n204\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.93.188:9031 --host=0.0.0.0\n245\t0.05GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/log_m",
+    "num_tasks": 10000,
+    "perf_metrics": [
+        {
+            "perf_metric_name": "tasks_per_second",
```
Comment (rkooo567) [inline release/release_logs/1.13.0/benchmarks/many_tasks.json:7]: LGTM

File: release/release_logs/1.13.0/microbenchmark.json [hunk 15]
```diff
@@ -0,0 +1,274 @@
...
+    "multi_client_put_calls_Plasma_Store": [
+        9662.197103077946,
+        184.95923834781362
+    ],
+    "multi_client_put_gigabytes": [
+        40.94456764236526,
+        1.4646681109150272
+    ],
+    "multi_client_tasks_async": [
+        33373.49615641964,
```
Comment (rkooo567) [inline release/release_logs/1.13.0/microbenchmark.json:75]: this has a huge regression 46643 ->  33373. cc @scv119 do you know anything about this?

**Review by rkooo567 — APPROVED** [review]
cc @scv119 it looks like many of microbenchmark has 10~20% regressions. Is it a known issue? @krfricke do you know if any test env has been changed?

**Pull request merged**